deepchecks/tabular/checks/distribution/train_test_feature_drift.py

```diff
@@ -107,6 +117,7 @@ def __init__(
         else:
             raise DeepchecksValueError('sort_feature_by must be either "feature importance" or "drift score"')
         self.n_top_columns = n_top_columns
+        self.categorical_drift_method = categorical_drift_method
```

Comment:
maybe validate the value in the init?

Reply:
in similar cases we also don't check on init so I don't think this case should be an exception